src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -1,4 +1,4 @@
-{% load i18n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
+{% load i18n l10n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
 
 <aside class="file" aria-label="{% trans "Bestand" %} {{ name }}">
     <div class="file__file">
```

Comment:
In order to correct the not-stretchy behaviour, I would surround the first flex-column with it's own class here so the icon can then be BOTH centered and stretched (so needs to be a nested Flexbox), but make sure to not use the 'icon' class or else CSS will start thinking this column is also an actual material-icon :-/
`<p class="file__symbol">.....`